From: Mohamad Al-Jaf <mohamadaljaf@gmail.com>
Subject: [PATCH 4/4] ext-ms-mf-pal-l2-1-0: Add new dll.
Message-Id: <20211126091906.443008-4-mohamadaljaf@gmail.com>
Date: Fri, 26 Nov 2021 04:19:06 -0500
In-Reply-To: <20211126091906.443008-1-mohamadaljaf@gmail.com>
References: <20211126091906.443008-1-mohamadaljaf@gmail.com>

Enable Windows 10 msmpeg2vdec.dll to run.
Chromium Embedded Framework is used by some applications
to display videos and requires msmpeg2vdec.dll.

Signed-off-by: Mohamad Al-Jaf <mohamadaljaf@gmail.com>
---
 dlls/apisetschema/apisetschema.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dlls/apisetschema/apisetschema.spec b/dlls/apisetschema/apisetschema.spec
index 11111111111..11111111111 100644
--- a/dlls/apisetschema/apisetschema.spec
+++ b/dlls/apisetschema/apisetschema.spec
@@ -1,3 +1,4 @@
+apiset ext-ms-mf-pal-l2-1-0 = kernelbase.dll
 apiset api-ms-win-appmodel-runtime-l1-1-2 = kernelbase.dll
 apiset api-ms-win-base-bootconfig-l1-1-0 = advapi32.dll
 apiset api-ms-win-base-util-l1-1-0 = advapi32.dll

-- 
2.37.2

